foundry.md

```diff
@@ -383,6 +385,97 @@ This rule takes `uint256` value using `#asWord(#range(LM, ARGSTART +Int 4, 32)`
      [priority(40)]
 ```
 
+#### `load` - Loads a storage slot from an address.
+
+```
+function load(address account, bytes32 slot) external returns (bytes32);
+```
+
+`call.load` will match when the `load` function is called at the [Foundry cheatcode address](https://book.getfoundry.sh/cheatcodes/#cheatcodes-reference).
+This rule then loads the storage slot identified by `#asWord(#range(LM, ARGSTART +Int 36, 32))` (referring to `slot` argument) from account `#asWord(#range(LM, ARGSTART +Int 4, 32))` (referring to `account`) and sets the return value to the loaded slot value.
+
+```k
+    rule [call.load]:
+         <k> CALL _ CHEAT_ADDR _VALUE ARGSTART _ARGWIDTH RETSTART RETWIDTH
+          => #loadAccount #asWord(#range(LM, ARGSTART +Int 4, 32))
+          ~> #foundryVmLoad ARGSTART RETSTART RETWIDTH ...
+         </k>
+         <localMem> LM </localMem>
+      requires CHEAT_ADDR ==Int #address(FoundryCheat)
+       andBool #asWord(#range(LM, ARGSTART, 4)) ==Int 1719639408 // selector ( "load(address,bytes32)" )
+      [priority(40)]
+
+
+    syntax KItem ::= "#foundryVmLoad" Int Int Int
+ // ----------------------------------------------------------
+    rule <k> #foundryVmLoad ARGSTART RETSTART RETWIDTH
+          => #setLocalMem RETSTART RETWIDTH #bufStrict(32, #lookup(STORAGE, #asWord(#range(LM, ARGSTART +Int 36, 32))))
+          ~> 1 ~> #push ...
+         </k>
+         <output>
+           _ => #bufStrict(32, #lookup(STORAGE, #asWord(#range(LM, ARGSTART +Int 36, 32))))
+         </output>
+         <account>
+             <acctID> ACCTID </acctID>
+			 <storage> STORAGE </storage>
+             ...
+         </account>
+         <localMem> LM </localMem>
```

Comment:
Same here. Look through other files in the repo to see the formatting used elsewhere.

Reply:
`git grep "<storage>"` will do it.

Reply:
Oh, I have a new vim config and forgot to enable the option to highlight tabs. Should now be fixed